### PR TITLE
Add CabinJS to Redaction Alternatives (closes #850)

### LINKED ISSUE #850: Thoughts on recommending Cabin for more sensitive and automated data Redaction?
In the docs here <https://getpino.io/#/docs/redaction> could we mention CabinJS?

https://cabinjs.com

CabinJS lets you use pino (I use it in production daily with pino) but it also scrubs a ton of sensitive information and cleans up logs.  See our README section here <https://cabinjs.com/#/?id=security-privacy-and-business-focused>.

Curious if I made a PR, would it be accepted, and if so, what would you like the verbiage to say and be placed?

### PR DESCRIPTION
**Pull request by niftylettuce**

### CODE REVIEW

File: docs/redaction.md [hunk 1]
```diff
@@ -131,3 +131,7 @@ It's extremely important that path strings do not originate from user input.
...
+
+## Alternatives
+
+Alternatively to Pino's native redaction, [CabinJS](https://cabinjs.com) may serve your needs out of the box.  CabinJS is a JavaScript logging package and middleware which automatically strips 1600+ sensitive field names, credit card numbers, BasicAuth headers, Social Security Numbers, JSON Web Tokens, API Keys, CSRF Tokens, Stripe Tokens, Passwords, Salts, Hashes, Bank Account Numbers, Bank Routing Numbers, and more.  It also reduces disk storage costs by converting Stream, Buffer, and ArrayBuffer log data into simplified descriptive objects.  Cabin also populates an `app` property in your log metadata which contains your packages name, version, node version, git hash, git tag, environment, hostname, and pid. If you use the widely popular [Passport](https://github.com/jaredhanson/passport) package, then it will even auto-add a user object for you to your log metadata. Lastly, it is a drop-in replacement and alternative to many existing logging solutions, including bunyan, and works great with pino plugged in out of the box.  Read more at <https://cabinjs.com>.
```
Comment (jsumners) [inline docs/redaction.md:137]: 1. Please line break at ~80 characters.
2. This reads as nothing more than an advertisement. Notice that the rest of this document introduces the feature by showing how to enable it and the effects redaction will have. Reading this text, I still don't know how I would use Cabin with Pino.

Comment (niftylettuce) [reply docs/redaction.md:137]: Ok cool I will patch and ping you back with example integration / code snippets added here.  Great points @jsumners - thank you very much.

Comment (mcollina) [reply docs/redaction.md:137]: Please remove the vast majority of the text and add an example. I think the first sentence is ok.

Comment (davidmarkclements) [reply docs/redaction.md:137]: This is very cool - can we add a note about performance characteristics of this approach. I’m assuming native redaction (fast-redact) will be significantly higher performance 

In future I’d love to see a way for cabin and fast redact to work together (or at least to implement the same concept on fast redact)

**Pull request closed without merging**